olad/OladHTTPServer.cpp

```diff
@@ -1025,20 +961,19 @@ void OladHTTPServer::DecodePortIds(const string &port_ids,
     }
 
     unsigned int device_alias, port;
-    if (!StringToInt(tokens[0], &device_alias) ||
-        !StringToInt(tokens[2], &port)) {
+    if (!StringToInt(tokens[0], &device_alias)
+        || !StringToInt(tokens[2], &port)) {
```

Comment:
Was this generated by eclipse? We generally put the logic operator and the end of the line